### DISCUSSION TITLE
attrs 默认小驼峰形式属性名会转成 kebabCase

### DISCUSSION THREAD

**Discussion by Gssen** [antvis/X6#1978] [Q&A] [answered]
attrs 默认小驼峰形式属性名会转成 kebabCase，怎么取消转换呢，比如想设置 viewBox

**Comment by NewByVector** [comment]
viewBox 已经有特殊处理了 https://github.com/antvis/X6/blob/master/packages/x6/src/view/attr.ts#L587

**Reply by Gssen** [reply]
https://codesandbox.io/s/vue-shape-forked-6n1w32?file=/src/App.vue
通过 addTools 方法添加工具，设置viewBox渲染出来还是会被转换，是我哪里用的不对吗
![image](https://user-images.githubusercontent.com/26129337/162652296-2510eee5-5fa7-4a43-8683-d14c01f86dfd.png)

**Comment by NewByVector** [comment, marked as answer]
这里是一个 bug，已经提 [issue](https://github.com/antvis/X6/issues/1993)
